### ISSUE TITLE
AttributeError: module 'zhon' has no attribute 'hanzi'

### ISSUE THREAD

**Issue by gaoxuezhao** [tsroten/zhon#31]
AttributeError: module 'zhon' has no attribute 'hanzi'

**Comment by coreylam** [comment]
same problem happened to me , i  add import statment in the `__init__.py` file and fix it .

FILE:  `site-packages/zhon/__init__.py`

CODE:

```python
"""Provides constants used in Chinese text processing."""

__version__ = '1.1.5'

from . import hanzi
from . import pinyin
from . import zhuyin
from . import cedict
```

**Comment by zuijiawoniu** [comment]
copy from somwhere.
`
import zhon.hanzi
or 
from zhon import hanzi
`

**Comment by MiuGod0126** [comment]
> copy from somwhere. `import zhon.hanzi or from zhon import hanzi`



> copy from somwhere. `import zhon.hanzi or from zhon import hanzi`

it works ! thank you!!!

**Comment by tsroten** [comment]
This should be fixed as of [v2.0.0](https://pypi.org/project/zhon/2.0.0/). Thanks for reporting this!
